packages/core/src/connectors/index.ts

```diff
@@ -0,0 +1,24 @@
+import { findConnectorByIdAndType, insertConnector } from '@/queries/connector';
+
+import * as AliyunDM from './aliyun-dm';
+import { ConnectorInstance } from './types';
+
+const connectors: ConnectorInstance[] = [AliyunDM];
+
+export const getConnectorById = (id: string): ConnectorInstance | undefined => {
```

Comment:
```suggestion
export const getConnectorById = (id: string): Optional<ConnectorInstance> => {
```

Reply:
didn't see the usage, maybe add it later?

Reply:
feels like an api in the admin console, get connector details.